### ISSUE TITLE
naming convention is pretty confusing

### ISSUE THREAD

**Issue by bongolegend** [paper-dragonfly/ttt_web#1]
https://github.com/paper-dragonfly/ttt_web/blob/d2d68188c34a4d795be5bcb9b0bc6ff0ab0e0f17/ttt_web/ttt_app.py#L2

**Comment by paper-dragonfly** [comment]
Yeah not ideal but can I leave it and then just have better naming in the future. I'm not sure what the conventions would be for the different parts. ttt_logic instead of backend?

**Comment by bongolegend** [comment]
no, you should keep the part that's unique. the `backend` part, not the `ttt` part.
